### ISSUE TITLE
Rolling back

### ISSUE THREAD

**Issue by brianatssat** [githubteacher/foundations-05-19-2015#4]
If you find that the newly branched code breaks production can you roll back to point before branches merged?

**Comment by githubteacher** [comment]
Yes, this is a great question! We will have a glimpse of these options later in the class and will cover them in greater detail tomorrow in the Intermediate class.

**Comment by githubteacher** [comment]
@brianatssat did you see the option to revert at the bottom of your pull request after you merged it?
![screenshot 2015-05-19 13 43 42](https://cloud.githubusercontent.com/assets/2132216/7709771/435d0340-fe2d-11e4-8b12-d9a3b3ffd642.png)
